Living_lab.md

```diff
@@ -0,0 +1,68 @@
+# Living lab
+
+ArXivDigest operates as a living lab platform. It means that third-party researchers can experiment with novel recommendation approaches ("recommendations AI"), and the generated suggestions will be shown to users of the arXivDigest service.  This allows for realistic, in situ evaluation with real users.  Further, user interactions can be utilized for improving recommendation and explanation capabilities.
+
+## Methodology
+
+ArXivDigest provides a broker infrastructure that connects *users* (people who signed up for the arXivDigest service) and *systems* (methods developed for providing content recommendations).  
+
+  * By using the service, users agree that their interest profiles, including the article recommendations they have been presented with and the implicit/explicit feedback they provided on each, would be made available to systems.
+  * Systems are given a 2.5 hour window each day to download new content once it has been published on arXiv, generate recommendations for any or all registered arXivDigest users, and upload these via the [arXivDigest API](/arxivdigest/api).
+  * The broker takes all recommendations created for a given user, interleaves them, and shows the top-k recommendations to the user.  See the section on [interleaving](#interleaving) for the specifics.
+  * System performance is monitored continuously over time. For system comparison, performance is measured during a designated evaluation period.
+
+
+## Components
+
+The evaluation framework consists of the following components:
+
+  * [arXivDigest API](/arxivdigest/api/)
+    - A RESTful service for experimental recommender systems to access article and user data and to upload personalized recommendations.
+  * [Scraper](scraper/)
+    - It continuously monitors the arXiv RSS feed and downloads new articles.
```

Comment:
Does not actually monitor continuously, but rather checks on a configured (daily) interval, don't know how specific we need to be though.

Reply:
Grouped all batch processes